projects/templates/index.html

```diff
@@ -38,15 +38,15 @@
 
          <div class="container-fluid container-xl d-flex align-items-center justify-content-between">
 
-            <a href="index.html" class="logo d-flex align-items-center">
+            <a href="home" class="logo d-flex align-items-center">
```

Comment:
`href="home"` 
home is a simple string and not a django url tag 
such as 
{% url 'home' %}